src/index.js

```diff
@@ -65,5 +68,8 @@ export default function(str) {
       this.traverse(loop);
     }
   });
-  return recast.print(ast).code;
+  const result = recast.print(ast, {
+    sourceMapName: 'map.json',
```

Comment:
Should we follow the convention of just adding `.map` to the filename?

Reply:
Good idea.